packages/redis-lwt/redis-lwt.0.3.4/opam

```diff
@@ -0,0 +1,15 @@
+opam-version: "1.2"
+maintainer: "aluuu@husa.su"
+authors: ["Mike Wells" "David Höppner" "Aleksandr Dinu"]
+homepage: "https://github.com/0xffea/ocaml-redis"
+bug-reports: "https://github.com/0xffea/ocaml-redis/issues"
+license: "BSD3"
+dev-repo: "https://github.com/0xffea/ocaml-redis.git"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+depends: [
+  "jbuilder" {build}
+  "redis"
```

Comment:
@avsm makes sense.

@rgrinberg are you talking about dependency on `redis` or is it reply to Anil?